<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vipwindow</class>
 <widget class="QDialog" name="vipwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>会员管理</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>921</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>70</y>
     <width>221</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>会员类型管理</string>
   </property>
   <widget class="QTableWidget" name="vipTypeTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>181</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>－</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>点击加号新建会员类型，或选择一项后按减号删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveAndExitButton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存并退出</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
   </property>
  </widget>
  <widget class="QPushButton" name="addVipButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>新会员</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
  </widget>
  <widget class="QPushButton" name="editVipButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑选中的会员信息</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteVipButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除会员</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListRemove"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="idSearch">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按会员卡号查询</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
